Documentation/data-cleanup/resource_delete.py

```
@@ -0,0 +1,219 @@
+                process_observation(solution_id)
+
+            collection_id = mb.get_collection_id(solution_id)
+            mb.delete_collection(collection_id)
+            list_of_groups = mb.get_permission_groups()
+            group_id = mb.get_permission_group_id(list_of_groups,solution_id)
+            mb.delete_permission_group(group_id)   
+
+        else:
+            logger.info("Skipping unsupported event type.")
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)